README.md

```diff
@@ -66,14 +66,14 @@ Check out the [full, working example repo](http://github.com/ethanresnick/json-a
 
 # Core Concepts
 ## Resource Type Descriptions <a name="resource-type-descriptions"></a>
-The JSON-API spec is built around the idea of typed resource collections. For example, you can have a `"people"` collection and a `"companies"` collection. (By convention, type names are plural and lowercase.)
+The JSON-API spec is built around the idea of typed resource collections. For example, you can have a `"people"` collection and a `"companies"` collection. (By convention, type names are plural, lowercase, and dasherized.)
 
 To use this library, you describe the special behavior (if any) that resources of each type should have, and then register that description with a central `ResourceTypeRegistry`. Then the library takes care of the rest. A resource type description is simply an object with the following properties:
 
 - `urlTemplates`: an object containing url templates used to output the json for resources of the type being described. Currently, the supported keys are: `"self"`, which defines the template used for building [resource urls](http://jsonapi.org/format/#document-structure-resource-urls) of that type, and `"relationship"`, which defines the template that will be used for [relationship urls](http://jsonapi.org/format/#fetching-relationships).
 - `dbAdapter`: the [database adapter](#database-adapters) to use to find and update these resources. By specifying this for each resource type, different resource types can live in different kinds of databases.
 
-- <a name="before-render"></a>`beforeRender` (optional): a function called on each resource after it's found by the adapter but before it's sent to the client. This lets you do things like hide fields that some users aren't authorized to see.
+- <a name="before-render"></a>`beforeRender` (optional): a function called on each resource after it's found by the adapter but before it's sent to the client. This lets you do things like hide fields that some users aren't authorized to see. It receives `resource`, `req`, `res`, and `superFn` as input and either a transformed resource or a promise as output. Returning or resolving to `undefined` will remove the resource from the response.
```

Comment:
No worries! I can clone your fork, cherry-pick your commit, and then edit it before merging, but the easiest thing is for you to push a new commit. You can even use rebase (since nothing depends on your fork yet) to merge your new commit into your last one, to make the history a bit neater, and then force-push that new version.

Something like:

1. `git commit -m "message here doesn't matter"`
2. `git rebase -i HEAD~2`, which'll select the last two commits for rebase, and then you can squash the second into the first and write a new commit message (or use the first commit's message) for the new commit.
3. `git push --force`

(Apologies if any of the above is remedial, or bad practice—git's intricacies are a bit of a mystery to me—but the above is what I usually do when making a PR, and it's never caused problems.)